source/includes/Terms and Conditions

```diff
@@ -0,0 +1,3 @@
+Use of any ReachLocal, Inc. (“ReachLocal”) Application Program Interface(s) (collectively, the “API Service”) is subject to the ReachLocal Inc. Online Marketing Services Terms and Conditions as set forth at www.reachlocal.com/tc (the “Marketing Services Terms”), including, without limitation, the Client Center Product Terms, and any other applicable agreement between you and ReachLocal. ReachLocal provides the API Service as an extension of the Client Center software and reserves the exclusive right to limit or revoke access to the API Service at any time and for any reason.
```

Comment:
this file needs to be named _terms_and_conditions.md.  Then you will make a change in https://raw.githubusercontent.com/reachlocal/api-docs/master/source/index.html.md to include this new file.  You probably want to put it right before the errors entry.

You will also want to create a heading inside of this file.  Something like 

# Terms And Conditions

On the first line of the file.